docs/README-develop-beaker-scripts.md

```diff
@@ -61,6 +67,8 @@ configure terminal
 end
 ~~~
 
+*Note: To enable sshd inside the `open agent container (OAC)` reference `OAC` documentation.
```

Comment:
Looks like there's a missing * at the end of this line (inherited from develop, but it's messing up the XR section).